### PR TITLE
luna version update

### PR DESCRIPTION
**Pull request by mendonk**
2.7 to 2.8

### CODE REVIEW

**Review by jbisso — APPROVED** [review]
Looks good; merge it.

**Comment by jgillenwater** [comment]
@mendonk Can we make a variable for the version number so we can change it in fewer places for the next revision?

cc/ @jbisso

**Comment by eolivelli** [comment]
@nicoloboschi do you want to double check?

File: modules/ROOT/pages/admin-console-tutorial.adoc [hunk 1]
```diff
@@ -57,7 +57,7 @@ See the <<codeSamples,Code samples>> section for a look at the clients.
 
 On the Pulsar Admin Console's Code Samples page, there are examples of Java, Python, Golang, Node.js, WebSocket, and HTTP clients.  Each example shows Producer, Consumer, and Reader code; plus language-specific examples of setting project properties and dependencies. For example with Java, if you are using Maven in your project, add the following to the `<properties>` section of your `pom.xml` file:
 
-`<pulsar.version>2.6.2</pulsar.version>`
+`<pulsar.version>2.8.0</pulsar.version>`
```
Comment (nicoloboschi) [inline modules/ROOT/pages/admin-console-tutorial.adoc:60]: we can point to 2.8.1

File: modules/ROOT/pages/admin-console-tutorial.adoc [hunk 2]
```diff
@@ -72,7 +72,7 @@ And add the following to the `pom.xml` file's `<dependencies>` section:
 If you are using Gradle, add this to your `build.gradle` file:
 
 ----
-def pulsarVersion = '2.6.2'
+def pulsarVersion = '2.8.0'
```
Comment (nicoloboschi) [inline modules/ROOT/pages/admin-console-tutorial.adoc:75]: we can point to 2.8.1

**Review by nicoloboschi — COMMENTED** [review]
LGTM, only a note: we should suggest to use Pulsar Client 2.8.1 which is the latest stable version of Apache Pulsar

**Pull request merged**